File: PRAKTIKUM PEMROGRAMAN I P1/MODUL 4/SOAL 4/PRAK404-2410817210002-Sebastian_Abe_Santoso.c
#include <stdio.h>
int main() {
    float NilaiPertama, NilaiKedua;
    int pilih;
    char NAMAANDA[50] = "Sebastian Abe Santoso";
    while (1) {
        printf("Pilih program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\n");
        printf("Masukkan Pilihan : ");
        scanf("%d", &pilih);
        if (pilih == 5) {
            printf("Terimakasih, telah menggunakan kalkulator %s\n", NAMAANDA);
            break;
        }
        if (pilih < 1 || pilih > 5) {
            printf("Input anda salah, silahkan coba lagi.\n");
            continue;
        }
        printf("Masukkan Nilai Pertama: ");
        scanf("%f", &NilaiPertama);
        printf("Masukkan Nilai Kedua : ");
        scanf("%f", &NilaiKedua);
        if (pilih == 1) {
            float Hasil = NilaiPertama + NilaiKedua;
            printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", NilaiPertama, NilaiKedua, Hasil);
        } else if (pilih == 2) {
            float Hasil = NilaiPertama - NilaiKedua;
            printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n", NilaiPertama, NilaiKedua, Hasil); 
        } else if (pilih == 3) {
            float Hasil = NilaiPertama * NilaiKedua;
            printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n", NilaiPertama, NilaiKedua, Hasil); 
        } else if (pilih == 4) {
            if (NilaiKedua != 0) { 
                float Hasil = NilaiPertama / NilaiKedua;
                printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n", NilaiPertama, NilaiKedua, Hasil); 
            } else {
                printf("Error: Tak Terdefinisi\n");
            }
        }
    } return 0;
}

File: MODUL 4/SOAL 4/PRAK404-2410817210002-Sebastian_Abe_Santoso.py
NAMAANDA = "Sebastian Abe Santoso"
while True:
    print("Pilih program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit")
    pilih = int(input("Masukkan Pilihan : "))
    if pilih == 5:
        print("Terimakasih, telah menggunakan kalkulator", NAMAANDA)
        break
    if pilih > 5 or pilih < 1:
        print("Input anda salah, silahkan coba lagi.")
        continue
    NilaiPertama = float(input("Masukkan Nilai Pertama: "))
    NilaiKedua = float(input("Masukkan Nilai Kedua: "))
    if pilih == 1:
        Hasil = NilaiPertama + NilaiKedua
        print(f"Hasil penjumlahan antara {NilaiPertama:.2f} dan {NilaiKedua:.2f} adalah {Hasil:.2f}")
    elif pilih == 2:
        Hasil = NilaiPertama - NilaiKedua
        print(f"Hasil pengurangan antara {NilaiPertama:.2f} dan {NilaiKedua:.2f} adalah {Hasil:.2f}")
    elif pilih == 3:
        Hasil = NilaiPertama * NilaiKedua
        print(f"Hasil perkalian antara {NilaiPertama:.2f} dan {NilaiKedua:.2f} adalah {Hasil:.2f}")
    elif pilih == 4:
        if NilaiKedua != 0:
            Hasil = NilaiPertama / NilaiKedua
            print(f"Hasil pembagian antara {NilaiPertama:.2f} dan {NilaiKedua:.2f} adalah {Hasil:.2f}")
        else:
            print("Error: Tak Terdefinisi (Pembagian dengan nol)")